README.md

```diff
@@ -40,6 +40,7 @@ necessary to verify the signature.
 * `options` `<Object>`
   * `fulcioURL` `<string>`: The base URL of the Fulcio instance to use for retrieving the signing certificate. Defaults to `'https://fulcio.sigstore.dev'`.
   * `rekorURL` `<string>`: The base URL of the Rekor instance to use when adding the signature to the transparency log. Defaults to `'https://rekor.sigstore.dev'`.
+  * `tsaURL` `<string>`: The base URL of the Timestamp Authority instance to use when requesting a signed timestamp. If omitted, no timestamp will be requested.
```

Comment:
Should we name this `tsaServerURL` to match cosign? https://github.com/sigstore/cosign/blob/cc969943428c109904d6a1888eb44c1576769cbd/cmd/cosign/cli/attest/attest.go#L83
```suggestion
  * `tsaServerURL` `<string>`: The base URL of the Timestamp Authority instance to use when requesting a signed timestamp. If omitted, no timestamp will be requested.
```

Reply:
Updated